app/models/manageiq/providers/google/cloud_manager/event_catcher/runner.rb

```diff
@@ -25,7 +25,8 @@ def monitor_events
 
   def queue_event(event)
     _log.info "#{log_prefix} Caught event #{parse_event_type(event)} for #{parse_resource_id(event)}"
-    EmsEvent.add_queue('add_google', @cfg[:ems_id], event)
+    event_hash = ManageIQ::Providers::Google::CloudManager::EventParser.event_to_hash(event, :ems_id)
```

Comment:
The old code passed the actual ems_id to event_to_hash, but the new code passes a symbol?  Is this a typo?